packages/smooth_app/lib/cards/product_cards/knowledge_panels/knowledge_panel_title_card.dart

```diff
@@ -83,6 +88,21 @@ class KnowledgePanelTitleCard extends StatelessWidget {
   }
 
   Color? _getColorFromEvaluation(Evaluation evaluation) {
```

Comment:
Never `null`: `Color` instead of `Color?`.

Reply:
Replaced `Color?` with `Color`.